### PR TITLE
god i want to eat strawberries now

### PR DESCRIPTION
**Pull request by ryanchou-dev**
_If any of the below don't apply to this Pull Request, mark the checkbox as done._

- [ ] I have tested my code.
- [ ] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [ ] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which include the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making the changes.

### CODE REVIEW

**Comment by SansPapyrus683** [comment]
uh could you follow the template i gave in my pr

**Comment by ryanchou-dev** [comment]
oops

File: solutions/silver/usaco-990.mdx [hunk 1]
```diff
@@ -2,20 +2,31 @@
 id: usaco-990
 source: USACO Silver 2020 January
 title: Berry Picking
-author: Qi Wang (Java), Daniel Suh (C++), Juheon Rhee (Python)
+author: Qi Wang, Daniel Suh, Juheon Rhee
 ---
 
+<Spoiler title="Hint 1">
+Let $k$ equal the maximum number of berries in one basket.
+What can we say about the capacity of the rest of the baskets?
```
Comment (SansPapyrus683) [inline solutions/silver/usaco-990.mdx:10]: i might just be rusty w/ this problem but
what does max # of berries? isn't it just the largest tree?

Comment (ryanchou-dev) [reply solutions/silver/usaco-990.mdx:10]: you don't have to take the entire tree ?

and motivation is that if you have a $k$-basket, then you should try to make as many $k$-baskets as possible.

Comment (SansPapyrus683) [reply solutions/silver/usaco-990.mdx:10]: hb uh
"say our baskets were [1,2,3,4,5]. is there any difference between this basket set and [1, 2, 3, 3, 3]?"
not sure if this encourages the mindset for the solution but

Comment (ryanchou-dev) [reply solutions/silver/usaco-990.mdx:10]: yeah that works. it does kinda give it away though. maybe we should leave drawing out the cases to the reader?

Comment (SansPapyrus683) [reply solutions/silver/usaco-990.mdx:10]: "is it _really_ always the best for bessie to take all the berries from the trees?"

Comment (ryanchou-dev) [reply solutions/silver/usaco-990.mdx:10]: ummm wait how does that relate

we want the reader to figure out that given an upper bound for each basket, we want as many baskets to reach this bound. the trees are just an interference in achieving this.

Comment (SansPapyrus683) [reply solutions/silver/usaco-990.mdx:10]: how would they come upon the idea for an upper bound though, that seems so jank

Comment (ryanchou-dev) [reply solutions/silver/usaco-990.mdx:10]: well if we give the maximum half of the baskets to bessie, then we want to other half to contain as much as possible, which implies that.

Comment (SansPapyrus683) [reply solutions/silver/usaco-990.mdx:10]: well, my point is when i read this hint at first it's just like "k is just the largest tree bruh"

Comment (ryanchou-dev) [reply solutions/silver/usaco-990.mdx:10]: oh ok, wdyt abt something like:

---

Let's say Bessie and Elsie get the same number of berries, what's a case that give us this? What can we do if this construction doesn't exist?

Comment (SansPapyrus683) [reply solutions/silver/usaco-990.mdx:10]: how can a construction suddenly not exist

Comment (ryanchou-dev) [reply solutions/silver/usaco-990.mdx:10]: Let's say Bessie and Elsie get the same number of berries, what's a case that give us this? How can we increase this value?

Comment (ryanchou-dev) [reply solutions/silver/usaco-990.mdx:10]: [oopsies i forgot we weren't fixing any values]

Comment (SansPapyrus683) [reply solutions/silver/usaco-990.mdx:10]: like bessie & elsie both get k berries?

Comment (ryanchou-dev) [reply solutions/silver/usaco-990.mdx:10]: yea

Comment (SansPapyrus683) [reply solutions/silver/usaco-990.mdx:10]: yeah that seems nice

**Pull request merged**